content/en/Integrations/IntegrationBuilder/How to Guides/import-assets.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Import assets from Google Sheets"
+linkTitle: "Import assets from Google Sheets"
+weight: 60
+---
+
+{{% pageinfo %}}
+How to use Integration Builder to import assets from Google Sheets.
+{{% /pageinfo %}}
+
+## Import assets from Google Sheets
+
+To import assets from Google Sheets, you need to follow two general steps:
+
+1. Create a spreadsheet in Google Sheets.
+2. Create a Google Sheets connection.
+3. Find and select the recipe
+4. Copy and customize the recipe
+5. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create a spreadsheet in Google Sheets
+
+To import assets from Google Sheets into the Cobalt platform, we first have to create a spreadsheet with the following columns: `Title`, `Type`, `Description`.
+
+`Title` and `Description` can be anything, but `Type` needs to be one of: `unknown`, `web`, `mobile`, `external_network`, `api`, `cloud_config`, `desktop_obsolete`, `internal_network`, `web_plus_api`, `web_plus_external_network`, `web_plus_mobile`, `wireless_network`, `iot`, `desktop`, `physical`, `other`, `web_plus_api_plus_external_network`, `internal_network_plus_external_network`, `mobile_plus_api`, `web_plus_mobile_plus_api`.
```

Comment:
Maybe the `Type` enumeration should use an unordered list? Semantically, it might even make sense to make it a description list.